docs/topics/mta-web-assessing-apps.adoc

```diff
@@ -4,55 +4,41 @@
 
 :_content-type: PROCEDURE
 [id="mta-web-assessing-apps_{context}"]
-= Assessing applications
+= Assessing an application by using the MTA assessment questionnaire
 
-You can use the {ProductFullName} to determine the risks involved in containerizing an application.
+You can use the default Legacy Pathfinder {ProductFullName} questionnaire to determine the risks involved in containerizing an application before reviewing the application. For more information, see xref:mta-default-questionnaire_user-interface-guide[The default assessment questionnaire].
 
-A built-in questionnaire is provided for assessing the risks involved in containerizing an application. For more details, see xref:mta-default-questionnaire_user-interface-guide[Default questionnaire].
+NOTE: To assess risks involved in containerizing applications, you can also create and use custom questionnaires. In this case, use the `Skipping assessment` option during the assessment. For more information, see xref:mta-custom-questionnaire_user-interface-guide[The custom assessment questionnaire].
```

Comment:
This statement needs to be removed since this is not accurate. In order to use custom questionnaires for assessment, the custom questionnaire needs to be marked as 'Required'.  The Assessment procedure is the same regardless of the questionnaire used (default or custom ) .

'In this case, use the `Skipping assessment` option during the assessment.'

Reply:
The application review procedure is duplicated in the following sections. 
6.3.3. Assessing an application by using the MTA assessment questionnaire
6.5. Reviewing an application 

1)I suggest that sections 6.5 and 6.5.1 be moved up right after 6.3.3.
2) I also recommend that the Review procedure be removed from section 6.3.3. by replacing 

   9. Click Save and Review to review the assessment.

   with 

   9. Click Save to review the assessmen.
3) In the 'Reviewing an application' section, state that there are two ways to review applications .
   1) One way is to click 'Save and Review' while assessing the application
   2) Second way is to click 'Review' from the application kebab menu.

Reply:
Attempted to rewrite in the next commit.

Reply:
@nachandr, please, check the changed version for both sections :-)

Reply:
Just reviewed the changed version for both section.  LGTM .